app/models/miq_request_task.rb

```diff
@@ -85,6 +85,11 @@ def update_request_status
     update_and_notify_parent(:state => req_state, :status => req_status, :message => display_message(msg))
   end
 
+  def update_request_task(values)
```

Comment:
Would `update_options` be a more descriptive name?

Reply:
@bdunne A single call allows us to filter which columns in the object is editable. Currently the only one that we allow updating is options. There are other columns like message and description that could be edited by the user. This is being done for update from the REST API and we dont want to keep adding new API's to edit the resource.